modules/serverless-event-delivery-component-behaviors.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// serverless/event_workflows/serverless-event-delivery.adoc
+
+[id="serverless-event-delivery-component-behaviors_{context}"]
+= Event delivery behavior for Knative Eventing components
+
+Different Knative Eventing components have their own behavior patterns that are followed for event delivery.
+
+== Delivery failure status codes
+
+The channel or broker receiver may respond with the following status codes if an event fails to be delivered:
+
+`500`:: This is a generic status code which means that the event was not delivered successfully.
+`404`:: This status code means that the channel or broker the event is being delivered to does not exist, or that the `Host` header is incorrect.
+`400`:: This status code means that the event being sent to the receiver is invalid.
+
+== Default channel and broker event delivery
```

Comment:
Is it abaut the default channel and the broke or about retries and dead-letter sink ? 

Maybe something like `== Common event deliver options` would be a better headline.